### ISSUE TITLE
Activation of power lines for peripherials on different boards

### ISSUE THREAD

**Issue by pjalocha** [pjalocha/ogn-tracker#1]
This thread is to discuss what power outputs should be turned on on various boards as to make the peripherials function properly.
With the power control chips AXP192, AXP202, AXP2101 each having several power outputs it is not trivial to know which power output to turn on and what voltage should bbe setup there.

**Comment by b3nn0** [comment]
Maybe this can serve as inspiration:
https://github.com/lyusupov/SoftRF/blob/20a23dd60283f88fcabdaaa9ace90719a5a8289a/software/firmware/source/SoftRF/src/platform/ESP32.cpp#L641
it seems to handle all the different AXP chips on the T-Beams.

**Comment by pjalocha** [comment]
Indeed, and the schematics fo various LilyGo modules: https://github.com/Xinyuan-LilyGO/LilyGo-LoRa-Series

**Comment by tfede37** [comment]
> This thread is to discuss what power outputs should be turned on on various boards as to make the peripherials function properly. With the power control chips AXP192, AXP202, AXP2101 each having several power outputs it is not trivial to know which power output to turn on and what voltage should bbe setup there.
> 
> Two librarries can be chosen bu -D in the platforrmio.ini and they support diferent power control chips WITH_AXP supports AXP192 and AXP202 WITH_XPOWERS supports AXP192 and AXP2101

So this means i should choose -DWITH_XPOWERS instead of -DWITH_AXP if i have the AXP2101 ?

Thanks

Federico

**Comment by pjalocha** [comment]
Yes, this XPOWERS librarry works for me, when it comes to AXP2101
